Libraries/CustomComponents/ListView/ListView.js

```diff
@@ -164,6 +164,13 @@ var ListView = React.createClass({
      */
     renderSectionHeader: PropTypes.func,
     /**
+     * (props) => renderable
+     *
+     * A function that returns the scroll view in which the list rows are
+     * rendered. Defaults to returning a ScrollView with the given props.
+     */
+    renderScrollView: React.PropTypes.func.isRequired,
```

Comment:
Also, I agree that "ScrollView" is a little confusing here - `renderScrollComponent` might be better.  Can you update?